### ISSUE TITLE
Search Core / Search API have an orphaned label

### ISSUE THREAD

**Issue by mgifford** [wet-boew/wet-boew-drupal#215]
Search Core / Search API drop-down has an accessibility issue. According to WAVE got an orphan label.

This may be a problem with WAVE, but it's reporting a problem with:
  <label for="edit-custom-search-paths">Customize your search </label>

I think the for should be removed and it should just be under the fieldset as per:
  http://www.w3.org/wiki/HTML/Elements/fieldset

We need a public space to test the distro.

**Comment by sylus** [comment]
Hey Mike, tonight or tomorrow morning I am going to setup a test space on wet.openplus.ca. Ideally soon after we can use the wet.coscoe.org domain :)

**Comment by sylus** [comment]
Also just in case people were curious have only committed a wetkit_search that deals with custom_search, the search api logic has not been added yet as trying to limit module creep. Need to evaluate whether should exist as a local or external app.

**Comment by sylus** [comment]
http://wet.openplus.ca is up

**Comment by sylus** [comment]
Fixed orphaned label
